src/client.cpp

```diff
@@ -99,7 +101,7 @@ ADDON_STATUS ADDON_Create(void* hdl, void* props)
     }
 
     /* Don't log trace messages unless told so */
-    if (level == LogLevel::LEVEL_TRACE && !Settings::GetInstance().GetTraceDebug())
```

Comment:
We introduced this setting for a reason. Just deleting the code is for sure not the solution for solving the problem not to have have a `CTvheadend` instance at hand here.